cloudify_types/cloudify_types/component/polling.py

```diff
@@ -0,0 +1,260 @@
+# Copyright (c) 2017-2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from os import getenv
+
+from cloudify import ctx
+from cloudify.exceptions import NonRecoverableError
+from cloudify_rest_client.exceptions import CloudifyClientError
+
+from .constants import POLLING_INTERVAL
+
+
+def any_bp_by_id(_client, _bp_id):
+    resource_type = 'blueprints'
+    return any_resource_by_id(_client, _bp_id, resource_type)
```

Comment:
I think it would be best if we rename these to `any_blueprint_by_id`, `any_deployment_by_id`.

Reply:
But you are fetching a blueprint, no?

Reply:
as in `bp` should be `blueprint` and `dp` should be `deployment`.